tests/aws/services/lambda_/test_lambda.py

```diff
@@ -829,16 +830,16 @@ class TestLambdaURL:
     # TODO: add more tests
 
     @pytest.mark.parametrize(
-        "returnvalue",
+        "returnvalue,custom_id",
```

Comment:
As it's a test marked with `aws.validated`, we would need to run against AWS just to confirm that it runs after being updated, and that will update the `test_lambda.validation.json` file as well.